test/package.json

```diff
@@ -21,7 +21,6 @@
     "react-dom": "^16.3.2",
     "react-hot-loader": "^1.3.0",
     "react": "^16.3.2",
-    "shelljs": "^0.7.7",
```

Comment:
Moved this to root `package.json` sine its used in the main root workspace and not the `test` workspace here